qt/applications/workbench/workbench/widgets/about/view.py

```diff
@@ -5,47 +5,327 @@
+from qtpy.QtWidgets import QVBoxLayout, QHBoxLayout, QSpacerItem, QSizePolicy, QLabel, QDialog, \
+    QGroupBox, QFormLayout, QComboBox, QPushButton, QCheckBox
 
-from mantidqt.utils.qt import load_ui
+REFERENCE_HEIGHT = 642
+REFERENCE_WIDTH = 745
 
-form, base = load_ui(__file__, "about.ui")
 
-
-class AboutView(base, form):
+class AboutView(QDialog):
     def __init__(self, parent, presenter, version_text, date_text = None):
         super(AboutView, self).__init__(parent)
-        self.setupUi(self)
-        self.set_layout(version_text, date_text)
+        self.background_pixmap = QPixmap(':/images/First_use_Background.png')
+        self.mantid_pixmap = QPixmap(':/images/mantid_smaller.png')
+        self.lbl_version = QLabel()
+        self.clb_release_notes = QCommandLinkButton()
+        self.clb_sample_datasets = QCommandLinkButton()
+        self.clb_mantid_introduction = QCommandLinkButton()
+        self.clb_python_introduction = QCommandLinkButton()
+        self.clb_python_in_mantid = QCommandLinkButton()
+        self.clb_extending_mantid = QCommandLinkButton()
+        self.cb_facility = QComboBox()
+        self.cb_instrument = QComboBox()
+        self.pb_manage_user_directories = QPushButton()
+        self.chk_allow_usage_data = QCheckBox()
+        self.lbl_privacy_policy = QLabel()
+        self.chk_do_not_show_until_next_release = QCheckBox()
+        self.pb_close = QPushButton()
+        self.setupUI()
+        self.customize_layout(version_text, date_text)
         self.presenter = presenter
 
-    def set_layout(self, version_text, date_text):
+    def paintEvent(self, event):
+        scaled_background = self.background_pixmap.scaled(self.rescale_w(self.background_pixmap.width()),
+                                                          self.rescale_h(self.background_pixmap.height()),
+                                                          Qt.KeepAspectRatio,
+                                                          Qt.SmoothTransformation)
+
+        scaled_mantid = self.mantid_pixmap.scaled(self.rescale_w(self.mantid_pixmap.width()),
+                                                  self.rescale_h(self.mantid_pixmap.height()),
+                                                  Qt.KeepAspectRatio,
+                                                  Qt.SmoothTransformation)
+        qp = QPainter()
+        qp.begin(self)
+        qp.drawPixmap(0, 0, scaled_background)
+        qp.drawPixmap(self.width() - scaled_mantid.width(), self.height()-scaled_mantid.height(), scaled_mantid)
+        qp.end()
+
+    def determine_dialog_dimensions(self):
+        width = REFERENCE_WIDTH
+        height = REFERENCE_HEIGHT
+
+        # gets the width of the screen where the main window is
+        screen = None
+        if hasattr(QGuiApplication,"screenAt"):
+            screen = QGuiApplication.screenAt(self.parent().geometry().center())
+        else:
+            screen = QGuiApplication.prmaryScreen()
```

Comment:
There's a typo here -> `QGuiApplication.primaryScreen()` that causes it to fail on startup.